modules/otel-config-multicluster.adoc

```diff
@@ -0,0 +1,160 @@
+// Module included in the following assemblies:
+// 
+// * otel/otel-configuring.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="gathering-observability-data-from-different-clusters_{context}"]
+= Gathering the observability data from different clusters with the OpenTelemetry Collector
+
+For a multicluster configuration, you can create one OpenTelemetry
+Collector instance in each one of the remote clusters and forward all the telemetry
+data to one OpenTelemetry Collector instance.
+
+.Prerequisites
+
+* The {OTELOperator} is installed.
+* The {TempoOperator} is installed.
+* A TempoStack is deployed on the cluster.
+
+.Procedure
+
+. Create a service account for the OpenTelemetry Collector.
++
+.Example ServiceAccount
+[source,yaml]
+----
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: otel-collector-deployment
+----
+
+. Create a cluster role for the service account.
++
+.Example ClusterRole
+[source,yaml]
+----
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: otel-collector
+rules:
+  <1>
+  <2>
```

Comment:
Not totally clear what these callouts are indicating. 

- Would there be a list for `k8sattributesprocessor` where `<1>` is and a list for `resourcedetectionprocessor` where `<2>` is? If so, is there example data that could be added here? 
- Or do both `<1>` and `<2>` apply to the `rules` stanza itself? If so, would suggest `rules:  <1>  <2>` (or combining them into a single callout).
- Or does it mean something else?

Reply:
It means the first option. I'll ask for some examples here for the next release. Thank you for your feedback.